src/main/java/seedu/address/model/summarybar/TotalApplicationsBox.java

```diff
@@ -0,0 +1,24 @@
+package seedu.address.model.summarybar;
+
+import java.util.Optional;
+
+/**
+ * Represents a SummaryBox that tracks the total number of applications in InternApply.
+ */
+public class TotalApplicationsBox extends SummaryBox {
```

Comment:
Naming of this class could follow TagSummaryBox.java

Maybe TotalApplicationsSummaryBox.java?